### PR TITLE
Chore/yarn upgrade

### PR DESCRIPTION
**Pull request by JonatanPaalsson**
OMEGA-X

## What is the reason for this change?
Upgrading packages

### What is the expected result?
No regression

## How is it solved?
running `yarn upgrade` and then manually upgrading some packages in order to solve issues

## Any changes/updates that may affect others?
|                                     |Changed|
|-------------------------------------|-------|
| Unit tests                          |   x   |
| Dependencies                        |    x   |
| Environment variables               |       |
| Build scripts                       |       |

### CODE REVIEW

**Comment by OysteinThuen** [comment]
Build #3006 successfully built and deployed.

- Prod: https://5f4df5d1f0854f993072d410--elvia-prod.netlify.app
- Test: https://5f4df5e9f0854f979872d508--elvia-test.netlify.app

**Comment by OysteinThuen** [comment]
Build #3064 successfully built and deployed.

- Prod: https://5f4e2efedb38d9d0dcf8182e--elvia-prod.netlify.app
- Test: https://5f4e2edd9d1fbe00b759a8b7--elvia-test.netlify.app

**Comment by OysteinThuen** [comment]
Build #3079 successfully built and deployed.

- Prod: https://5f4e37c7f747cc0106647d3e--elvia-prod.netlify.app
- Test: https://5f4e3785f747cc011e647d9e--elvia-test.netlify.app

**Comment by OysteinThuen** [comment]
Build #3088 successfully built and deployed.

- Prod: https://5f4e3e1f2ba4e007c9940762--elvia-prod.netlify.app
- Test: https://5f4e3e2152fcbe0831e9e992--elvia-test.netlify.app

**Comment by JonatanPaalsson** [comment]
This PR upgrades most of our packages so rigorous testing is required

File: packages/web/package.json [hunk 1]
```diff
@@ -76,20 +76,20 @@
...
+    "babel-jest": "^25.2.7",
     "babel-preset-gatsby": "^0.2.27",
     "before-build-webpack": "^0.2.9",
     "jest": "^25.2.7",
     "node-sass": "^4.13.0",
     "react-test-renderer": "^16.12.0",
     "redux-devtools-extension": "^2.13.8",
     "start-server-and-test": "^1.10.6",
-    "ts-jest": "^24.3.0",
+    "ts-jest": "^26.3.0",
```
Comment (t0ff3r) [inline packages/web/package.json:92]: This is a jump in versions

Comment (JonatanPaalsson) [reply packages/web/package.json:92]: Yes, this was needed when upgrading a minor for the testing-library

**Review by t0ff3r — APPROVED** [review]

**Pull request merged**